src/chart/bar/BarSeries.ts

```diff
@@ -125,6 +127,8 @@ class BarSeriesModel extends BaseBarSeriesModel<BarSeriesOption> {
 
         roundCap: false,
 
+        colorBy: 'seriesName',
```

Comment:
It's not necessary to write `colorBy: 'seriesName'` explicitly in series that should use the default colorBy strategy.